analytics_data_api/management/commands/generate_fake_course_data.py

```diff
@@ -36,7 +38,7 @@ class Command(BaseCommand):
         make_option('-n', '--num-weeks', action='store', type="int", dest='num_weeks',
                     help='Number of weeks worth of data to generate.'),
         make_option('-c', '--course_id', action='store', type='string', dest='course_id',
-                    default='edX/DemoX/Demo_Course', help='Course ID for which to generate fake data'),
+                    default='course-v1:edX+DemoX+Demo_Course', help='Course ID for which to generate fake data'),
```

Comment:
@dsjen I'm not sure about this change. I'm pretty confused by edX course IDs in general... do you think most developers (including OpenEdX) would have the new `course-v1` ID over the old `/` delimited one? I had to change it to this to get it to work for me.